packages/application-components/src/components/modal-pages/form-modal-page/form-modal-page.tsx

```diff
@@ -1,24 +1,14 @@
 import React from 'react';
 import { MessageDescriptor } from 'react-intl';
 import buttonMessages from '../../../utils/button-messages';
-import ModalPage from '../internals/modal-page';
-import ModalPageHeader from '../internals/modal-page-header';
-import ModalPageHeaderDefaultControls from '../internals/modal-page-header-default-controls';
-import { ContentWrapper } from '../internals/modal-page.styles';
+import CustomFormModalPage from '../custom-form-modal-page';
+import {
+  ConfirmButton,
+  CancelButton,
+} from '../../../utils/default-form-buttons';
```

Comment:
We should use `CustomFormModalPage.ConfirmButton` and `CustomFormModalPage.CancelButton` right?